src/schema/resolvers.js

```diff
@@ -1,8 +1,38 @@
+const _ = require('lodash');
 const pubsub = require('../pubsub');
 const fetch = require('node-fetch');
 
-function buildTopicFilters({OR = [], address, status}) {
-  const filter = (address || status) ? {} : null;
+/* Default limit number for query page */
+const DEFAULT_LIMIT_NUM = 50;
+const DEFAULT_SKIP_NUM = 0;
+
+/**
+ * Returns an options object to pass in database query
+ * @param  {Array} orderBy Array of Order type defined in schema
+ * @param  {Number} limit   limit number in paging; default value DEFAULT_LIMIT_NUM
+ * @param  {Number} skip    skip number in paging; default value DEFAULT_SKIP_NUM
+ * @return {Object}         
+ */
+function buildQueryOptions(orderBy, limit, skip) {
```

Comment:
cant find where this func is used in `resolver`

Reply:
removed entire method